src/disclosures/js/views/metric-view.js

```diff
@@ -5,94 +5,113 @@ var getSchool = require( '../dispatchers/get-school-values' );
 var formatUSD = require( 'format-usd' );
 
 var metricView = {
+  metrics: {
+    debtBurden: {
+      school: NaN,
```

Comment:
I have a few guesses, but rather than assume, 😄  I'll ask: why are we explicitly initializing these as NaNs?